### PR TITLE
init project

### PR DESCRIPTION
**Pull request by QuangHieuu**
https://edu-redmine.framgia.vn/issues/2639

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        <activity android:name=".MainActivity">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN"/>
+
+                <category android:name="android.intent.category.LAUNCHER"/>
+            </intent-filter>
+        </activity>
+    </application>
+
+</manifest>
```
Comment (phuongohn-1626) [inline app/src/main/AndroidManifest.xml:21]: thêm dòng cuối

File: app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
+    <background android:drawable="@drawable/ic_launcher_background"/>
+    <foreground android:drawable="@drawable/ic_launcher_foreground"/>
+</adaptive-icon>
```
Comment (phuongohn-1626) [inline app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml:5]: add blank

File: app/build.gradle [hunk 3]
```diff
@@ -0,0 +1,30 @@
+apply plugin: 'com.android.application'
+
+android {
+  compileSdkVersion 27
+  defaultConfig {
+    applicationId "com.music"
+    minSdkVersion 23
```
Comment (tuanlvt-1117) [inline app/build.gradle:7]: chuyển min thành 16 e

File: app/build.gradle [hunk 4]
```diff
@@ -0,0 +1,30 @@
+apply plugin: 'com.android.application'
+
+android {
+  compileSdkVersion 27
+  defaultConfig {
+    applicationId "com.music"
```
Comment (tuanlvt-1117) [inline app/build.gradle:6]: rename : com.framgia.music_40

File: app/src/androidTest/java/com/music/ExampleInstrumentedTest.java [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+@RunWith(AndroidJUnit4.class)
+public class ExampleInstrumentedTest {
+    @Test
+    public void useAppContext() {
+        // Context of the app under test.
+        Context appContext = InstrumentationRegistry.getTargetContext();
+
+        assertEquals("com.music", appContext.getPackageName());
```
Comment (tuanlvt-1117) [inline app/src/androidTest/java/com/music/ExampleInstrumentedTest.java:24]: above

File: app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml [hunk 6]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
+    <background android:drawable="@drawable/ic_launcher_background"/>
+    <foreground android:drawable="@drawable/ic_launcher_foreground"/>
+</adaptive-icon>
```
Comment (tuanlvt-1117) [inline app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml:5]: enter end line

File: app/src/main/res/values/dimens.xml [hunk 7]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="dp_50">50dp</dimen>
+    <dimen name="dp_20">20dp</dimen>
+    <dimen name="dp_250">250dp</dimen>
+</resources>
```
Comment (tuanlvt-1117) [inline app/src/main/res/values/dimens.xml:6]: enter end line

File: app/src/test/java/com/music/ExampleUnitTest.java [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class ExampleUnitTest {
+    @Test
+    public void addition_isCorrect() {
+        assertEquals(4, 2 + 2);
+    }
+}
```
Comment (tuanlvt-1117) [inline app/src/test/java/com/music/ExampleUnitTest.java:17]: enter end line

**Pull request closed without merging**